database/migrations/2023_02_01_134527_add_epic_id_to_stories_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddEpicIdToStoriesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('stories', function (Blueprint $table) {
+            $table->bigInteger('epic_id');
+            $table->foreign('epic_id')->references('id')->on('epics');
```

Comment:
Sempre meglio aggiungere il comportamento della relazione in caso di eliminazione dell'elemento padre: in questo caso si può prevedere l'eliminazione a cascata di tutte le storie che appartengono ad una EPICA che viene eliminata:

```
$table->foreign('epic_id')->references('id')->on('epics')->onDelete('cascade');
```